js_functions/task2.js

```diff
@@ -0,0 +1,14 @@
+const isPersonAdult = (age) => {
+if (age >= 18) {
+        return true;
+    } else if (age < 18) {
+        return false;
+    } else {
+        return errorMessage;
+    }
+}
+
+const errorMessage = 'It`s not a number'
```

Comment:
it works but I would expect to see initialization of `errorMessage` before it is used in the function

Reply:
But you do not need it in this task